### ISSUE TITLE
How do you know if it's right without training?

### ISSUE THREAD

**Issue by yjhdhr** [kyegomez/PALM-E#7]
THX. BUT...How do you know if your code is right without training?

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/kyegomez) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/kyegomez/PALM-E/issues/7">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/kyegomez/PALM-E/issues/7/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/kyegomez/PALM-E/issues/7/pledge.svg">
</picture>
</a>

**Comment by kyegomez** [comment]
@yjhdhr Great question: the output tensors will be within 0.0 and 1.0 denoting the logits which are transformed from random integers by layers of attention.
